### PR TITLE
Made fonts properties

### PR DESCRIPTION
**Pull request by jmbiggs**
I made the fonts properties so that they can be easily set/changed to custom values.

### CODE REVIEW

**Comment by gebeleysis** [comment]
Hey,
Thanks for submitting this pull request. I'm sorry for not seeing it before. I realise this was probably a bummer. Didn't get a notification :(
I'm not a github expert, but it could be that setting an assignee or a reviewer to the PR would have notified me.

I've got a couple of concerns about the PR though.

**Comment by gebeleysis** [comment]
Please see comment in code review. Assign the PR back to me when you're done.

Once again, thanks for your contribution. This project hasn't received much love recently but I'll try to be better at it in the future :)

**Comment by jmbiggs** [comment]
Oh, no problem.  I think you're right, I should have set an assignee or reviewer.. I'm still kind of new to GitHub's interface for pull requests.

I used this on an app that I'm not even working on anymore, but I'm happy to contribute!

File: RCTagProcessing/RCTagProcessing/Classes/RCTagProcessor.m [hunk 1]
```diff
@@ -22,6 +22,18 @@ + (instancetype)defaultInstance {
...
 
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        // set default fonts
+        _regularFont = [UIFont systemFontOfSize:10.42];
+        _boldFont = [UIFont boldSystemFontOfSize:10.42];
+        _smallFont = [UIFont systemFontOfSize:5.42];
+    }
```
Comment (jmbiggs) [inline RCTagProcessing/RCTagProcessing/Classes/RCTagProcessor.m]: Ah, I see.  The way I was using it was by creating one static instance of RCTagProcessor, but I'll take a look at this.

**Comment by jmbiggs** [comment]
Well, it appears that I don't have permission to reassign the pull request, but I pushed a new commit.

**Comment by gebeleysis** [comment]
Cool. Code looks good, all tests pass, there's no significant performance impact. Merged.
Thank you!

**Pull request merged**